[FW][FIX] stock_voucher_ux: print and assign method

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/stock#583

mav-adhoc committed:
[FIX] stock_voucher_ux: print and assign method

X-original-commit: 2308df474b879518373c69912e5a4a6e0329975a

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/stock/pull/604).

fw-bot-adhoc commented:
@mav-adhoc @jcadhoc cherrypicking of pull request ingadhoc/stock#583 failed.

stdout:
```
CONFLICT (modify/delete): stock_voucher_ux/wizards/stock_print_stock_voucher_views.xml deleted in HEAD and modified in 47f7c057 ([FIX] stock_voucher_ux: print and assign method). Version 47f7c057 ([FIX] stock_voucher_ux: print and assign method) of stock_voucher_ux/wizards/stock_print_stock_voucher_views.xml left in tree.
CONFLICT (modify/delete): stock_voucher_ux/wizards/stock_print_stock_voucher.py deleted in HEAD and modified in 47f7c057 ([FIX] stock_voucher_ux: print and assign method). Version 47f7c057 ([FIX] stock_voucher_ux: print and assign method) of stock_voucher_ux/wizards/stock_print_stock_voucher.py left in tree.
Auto-merging stock_voucher_ux/controllers/main.py
CONFLICT (content): Merge conflict in stock_voucher_ux/controllers/main.py
Auto-merging stock_voucher_ux/__manifest__.py
CONFLICT (content): Merge conflict in stock_voucher_ux/__manifest__.py

```

stderr:
```
16:26:03.115669 git.c:444               trace: built-in: git cherry-pick 47f7c057ee47b5c1a092b9f37b10e49a862577a1
error: could not apply 47f7c057... [FIX] stock_voucher_ux: print and assign method
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
